helm/values.yaml

```diff
@@ -1,96 +1,61 @@
-  name: ""
-
-podSecurityContext: {}
-  # fsGroup: 2000
-
-securityContext: {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
-
-service:
-  type: ClusterIP
-  port: 80
-
-ingress:
-  enabled: enable
-  className: ""
-  annotations:
-    cert-manager.io/cluster-issuer: letsencrypt-http01
-    external-dns.alpha.kubernetes.io/target: traefik-external.parity-chains.parity.io.
-    kubernetes.io/ingress.class: traefik-external
-    traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
-    traefik.ingress.kubernetes.io/router.tls: "true"
-  hosts:
-    - host: try-runtime.parity.io
-      paths:
-        - path: /
-          pathType: ImplementationSpecific
-  tls:
-    - secretName: try-runtime.parity.io
-      hosts:
-        - try-runtime.parity.io
-
-# We only want one instance at all times so that the execution queue can be
-# sanely tracked across multiple MRs.
-autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 1
-
-nodeSelector: {}
-
-affinity:
-  nodeAffinity:
-    requiredDuringSchedulingIgnoredDuringExecution:
-      nodeSelectorTerms:
-      - matchExpressions:
-        - key: cloud.google.com/gke-nodepool
-          operator: In
-          values:
-          - try-runtime-pool
-
-tolerations:
-- key: "app"
-  operator: "Equal"
-  value: "try-runtime"
-  effect: "NoExecute"
-
-env:
-  GITLAB_PUSH_NAMESPACE: parity/mirrors
-  GITLAB_JOB_IMAGE: paritytech/ci-linux:production
-  GITLAB_DOMAIN: gitlab.parity.io
-  PIPELINE_SCRIPTS_REPOSITORY: https://github.com/paritytech/command-bot-scripts/
-
-persistence:
-  enabled: true
-  mountPath: ""
-  subPath: ""
-  annotations: {}
-  storageClass: hdd-csi
-  size: 100G
-  accessModes:
-    - ReadWriteOnce
+# gcloud-sqlproxy:
+#   serviceAccountName: "command-bot"
+#   serviceAccount:
+#     create: true
+#     name: command-bot
+#   networkPolicy:
+#     enabled: false
+
+common:
+  fullnameOverride: "command-bot"
+  extraLabels:
+    team: "opstooling"
+  serviceAccount:
+    create: false
+  image:
+    repository: paritytech/command-bot
+  envFrom:
+    - secretRef:
+        name: command-bot
+  service:
+    ports:
+      - name: http
+        protocol: TCP
+        port: 80
+        targetPort: 3000
+  ingress:
+    enabled: true
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-dns01
+      kubernetes.io/ingress.class: traefik-external
+      traefik.ingress.kubernetes.io/router.entrypoints: web,websecure
+      traefik.ingress.kubernetes.io/router.tls: "true"
+  persistence:
+    enabled: true
+    mountPath: ""
+    subPath: ""
+    annotations: {}
+    storageClass: hdd-csi
+    size: 100G
+    accessModes:
+      - ReadWriteOnce
+  env:
+    GITLAB_PUSH_NAMESPACE: parity/mirrors
+    GITLAB_JOB_IMAGE: paritytech/ci-linux:production
+    GITLAB_DOMAIN: gitlab.parity.io
+    PIPELINE_SCRIPTS_REPOSITORY: https://github.com/paritytech/pipeline-scripts
```

Comment:
```suggestion
    PIPELINE_SCRIPTS_REPOSITORY: https://github.com/paritytech/command-bot-scripts/
```

Reply:
added in https://github.com/paritytech/command-bot/pull/149/commits/a0e119a01d8b1dc9888b86855cf38502cd2c08cb